Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -87,6 +89,20 @@ public object CreateInstance(Type type)
             return _appHost.CreateInstance(type);
         }
 
+        private string NormalizeUrlPath(string path)
```

Comment:
this should be `static` as it doesn't access any variables in `this`

Reply:
Ack.